src/ft2font.cpp

```diff
@@ -222,9 +222,9 @@ FT2Font::~FT2Font()
     close();
 }
 
-void FT2Font::open(FT_Open_Args &open_args)
+void FT2Font::open(FT_Long face_index, FT_Open_Args &open_args)
```

Comment:
flip the order here (and the call sights) to match the order of the FT call?

Reply:
Sure.